src/components/desktop/DesktopLayout.js

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components'
+
+import MapPage from '../map/MapPage'
+import ResizablePane from '../ui/ResizablePane'
+import Header from './Header'
+
+const DesktopContainer = styled.div`
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+`
+
+const ContentContainer = styled.main`
+  position: relative;
+  flex: 1;
+`
+
+const MapContainer = styled.div`
+  position: absolute;
+  right: 0;
+  height: 100%;
+  width: calc(100% - 310px);
+`
+
+const DesktopLayout = () => (
+  <DesktopContainer>
+    <Header />
+    <ContentContainer>
+      <ResizablePane minWidth={310} maxWidth="21.5vw" $defaultWidth="21.5vw" />
```

Comment:
Small thing, but maybe consider extracting these props into consts or a single const object? Not sure if it would be cleaner or not...

Reply:
I had to switch the side panel from being an "overlay" to an actual pane, so this code got updated. Did end up using constants for this because I had to kind of hack in the max size to work with the library I'm using. You can see in the latest commit.